model/Software/Properties/fileType.md

```diff
@@ -0,0 +1,16 @@
+# fileType
+
+## Summary
+
+TODO
+
+## Description
+
+A fileType is TODO
+
+## Metadata
+
+- name: fileType
+- Nature: DataProperty
+- Range: FileTypeVocab
```

Comment:
Is range FileTypeVocab? As the range of fileType is FileType[0..1] !!

Reply:
Yes, every enumeration (table at the bottom of the picture) will be a vocabulary. But we will be changing these FileTypes from what we've discussed last Tuesday.

Reply:
I mean the range should also be something like CustomDtypeVocab[0..1], or it is implicit from the class information?